examples/gno.land/p/demo/ownable/exts/authorizable/authorizable.gno

```diff
@@ -0,0 +1,70 @@
+// Package authorizable is an extension of p/demo/ownable;
+// It allows the user to instantiate an Authorizable struct, which extends
+// p/demo/ownable with a list of users that are authorized for something.
+// By using authorizable, you have a superuser (ownable), as well as another
+// authorization level, which can be used for adding moderators or similar to your realm.
+package authorizable
+
+import (
+	"std"
+
+	"gno.land/p/demo/ownable"
+)
+
+type Authorizable struct {
+	*ownable.Ownable
+	authorized map[std.Address]struct{}
```

Comment:
I suggest staying with `avl.Tree`, for now.

Reply:
Why? I wanted to have an example using map, since there are so few of them currently